contentcuration/contentcuration/models.py

```diff
@@ -833,8 +833,10 @@ class ContentNode(MPTTModel, models.Model):
                                    help_text=_("Ascending, lowest number shown first"))
     copyright_holder = models.CharField(max_length=200, null=True, blank=True, default="",
                                         help_text=_("Organization of person who holds the essential rights"))
-    cloned_source = TreeForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True, related_name='clones')
+    # legacy field...
     original_node = TreeForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True, related_name='duplicates')
+    cloned_source = TreeForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True, related_name='clones')
```

Comment:
@rtibbles Can you confirm that `original_node` and `cloned_source` are not used anywhere (legacy form the first initial draft design of content-provendance approach).

That was my impression from past reading of the code. If this is the case then we could think about removing these to columns to simplify the data model a bit.  [one time I spent hours debugging somethign accessing the `original_node_id` property when in fact what I really wanted was `original_source_node_id`]

Reply:
I can try to confirm this - although I've not been in the node provenance loop. I think @jamalex might have more context to confirm or deny.